doc/source/conf.py

```diff
@@ -25,6 +25,16 @@
 sys.path.insert(0, str(Path("../..").resolve()))
 sys.path.insert(0, str(Path("../../src").resolve()))
 
+# Here provide the default values for the Python version range
+PYTHON_MAX_VERSION = 3.13
```

Comment:
@sbillinge you are right. Adding `DEFAULT` here only causes confusion because this `PYTHON_MAX_VERSION = 3.13` will be used throughout the doc, not just for our CI workflows. So I removed the mention of `DEFAULT`. Thank you.